srml/staking/src/benches.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Parity Technologies
```

Comment:
Please switch to https://crates.io/crates/criterion

Reply:
I started with that (and already wrote some benches for storage using it) but had some issues placing the benchmarks inside the `src/` folder so that it can see the internal and private functions. afaik Criterion either does not support this (and only works with a separate `benches` folder in which you can bench your crate __externally__) or it was not worth the effort for me to configure it. 

With the helper macro, I don't see a significant benefit in using criterion.